x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/steps/blocks/action/processor_field_selector.tsx

```diff
@@ -73,7 +73,7 @@ export const ProcessorFieldSelector = ({
 
   const defaultLabel = i18n.translate(
     'xpack.streams.streamDetailView.managementTab.enrichment.processor.fieldSelectorSourceLabel',
-    { defaultMessage: 'Source field' }
+    { defaultMessage: 'Field' }
```

Comment:
This will change it for all fields 
I am not sure if that's what we'd want. Maybe it is? 

Only the `set` processor has this problem, the others could in theory stay with `Source Field` 

For Grok, Rename, Append, "source field" makes more sense, instead of just "field"

Reply:
Let's leave it as Source field then. I will change it, thanks.